docs/machine-learning/tutorials/image-classification-model-builder.md

```diff
@@ -0,0 +1,247 @@
+---
+title: 'Tutorial: Train an image classification model in Azure using Model Builder'
+description: Learn how to train an image classification model to recognize land use from satellite images in Azure using Model Builder
+author: luisquintanilla
+ms.author: luquinta
+ms.date: 09/21/2021
+ms.topic: tutorial
+ms.custom: mvc,mlnet-tooling
+#Customer intent: As a non-developer, I want to use Model Builder to automatically train a model in Azure to classify images using Model Builder.
+---
+
+# Train an image classification model in Azure using Model Builder
+
+Learn how to train an image classification model in Azure using Model Builder to categorize land use from satellite images.
+
+This tutorial shows you how to create a Universal Windows Platform (UWP) application that hosts a model in an C# ASP.NET Core Web API to categorize land use based on satellite images.
+
+In this tutorial, you:
+
+> [!div class="checklist"]
+>
+> - Prepare and understand the data
+> - Create a Model Builder config file
+> - Choose a scenario
+> - Load the data
+> - Create an experiment in Azure
+> - Train the model
+> - Evaluate the model
+> - Consume the model
+
+> [!NOTE]
+> Model Builder is currently in Preview.
+
+## Prerequisites
+
+- For a list of pre-requisites and installation instructions, visit the [Model Builder installation guide](https://docs.microsoft.com/dotnet/machine-learning/how-to-guides/install-model-builder).
+- Azure account. If you don't have one, [create a free Azure account](https://aka.ms/AMLFree).
+- ASP.NET and web development workload.
+
+## Model Builder image classification overview
+
+This sample creates a UWP application that categorizes land use from map satellite imagery using a deep learning model trained on Azure with Model Builder. The model itself is hosted as a web service in an ASP.NET Core Web API. You can find the source code for this tutorial in the [dotnet/machinelearning-samples](https://github.com/dotnet/machinelearning-samples/tree/master/samples/modelbuilder/ImageClassification_Azure_LandUse) GitHub repository
+
+## Create solution
+
+1. In Visual Studio, select **File > New > Project** from the menu bar.
+1. In the New Project dialog, type "Solution" into the search box.
+1. Select the **Blank Solution** template, then select the **Next** button.
+1. In the **Name** text box, type "LandUse".
+1. Select **Create**.
+
+## Create a Console Project
+
+1. In Solution Explorer, right-click the **LandUse** solution, then select **Add > New Project**.
```

Comment:
Since we're no longer working with solutions, you can remove the adding of a new project to the solution and instead you can create the project directly.